src/lib/components/Navigation/Navigation.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import { Meta } from "@storybook/react";
+
+import { Navigation } from "./Navigation";
+
+const meta: Meta<typeof Navigation> = {
+  title: "Components/Navigation",
```

Comment:
I feel that a `Sections` category would be more appropriate here (following the logic of increasing levels of complexity - e.g. link being an element, Navigation item a component, and navigation a section).
https://canonical.github.io/maas-react-components/?path=/docs/readme--docs#categories